packages/camera/camera/example/integration_test/camera_test.dart

```diff
@@ -240,4 +240,34 @@ void main() {
     },
     skip: !Platform.isAndroid,
   );
+
+  testWidgets(
+    'Image streaming persistence test on iOS',
+    (WidgetTester tester) async {
+      final List<CameraDescription> cameras = await availableCameras();
+      if (cameras.isEmpty) {
+        return;
+      }
+
+      final CameraController controller = CameraController(
+        cameras[0],
+        ResolutionPreset.max,
+        enableAudio: false,
+      );
+
+      await controller.initialize();
+      int _frame = 0;
+
+      await controller.startImageStream((CameraImage image) {
+        _frame++;
+      });
+
+      await Future.delayed(Duration(seconds: 5));
+      expect(_frame > 30, true);
```

Comment:
It seems like this is testing round-trip speed on the device it's running on, rather than anything about the code itself. Where is the value of `30` coming from? How do you know that this will always pass?

Reply:
Surely 30 is not important, so I removed it.
I wanted to test if I could keep it running for a certain amount of time without crashing.
I'm not sure if 5 seconds is a reasonable number in this case.
Actually, I don't know how to create such environment-dependent tests.